ipaserver/install/krbinstance.py

```diff
@@ -377,6 +381,15 @@ def setup_pkinit(self):
         # have any selinux issues with the file context
         shutil.copyfile(paths.IPA_CA_CRT, paths.CACERT_PEM)
 
+        # Now modify configuration to add pkinit anchors and restart KDC
+        self.sub_dict['NOPK'] = ''
+        self.__template_file(paths.KRB5KDC_KDC_CONF, chmod=None, backup=False)
+        try:
+            self.stop()
+            self.start()
+        except Exception:
+            root_logger.critical("krb5kdc service failed to restart")
```

Comment:
If this is critical error, please re-raise to stop installation altogether. If it is a soft error, please log at error level not critical.